client/src/app/site/motions/components/statute-paragraph-list/statute-paragraph-list.component.html

```diff
@@ -0,0 +1,87 @@
+<os-head-bar appName="Statute paragraphs" [plusButton]=true (plusButtonClicked)=onPlusButton()
+[menuList]="menuList" (ellipsisMenuItem)=onEllipsisItem($event)></os-head-bar>
+<div class="head-spacer"></div>
+<mat-card *ngIf="statuteParagraphToCreate">
+    <mat-card-title translate>Create new statute paragraph</mat-card-title>
+    <mat-card-content>
+        <form [formGroup]="createForm" (keydown)="keyDownFunction($event)">
+            <p>
+                <mat-form-field>
+                    <input formControlName="title" matInput placeholder="{{'Title' | translate}}" required>
+                    <mat-hint *ngIf="!createForm.controls.title.valid">
+                        <span translate>Required</span>
+                    </mat-hint>
+                </mat-form-field>
+            </p>
+            <p>
+                <mat-form-field>
+                    <textarea formControlName="text" matInput placeholder="{{'Statute paragraph' | translate}}" required></textarea>
+                    <mat-hint *ngIf="!createForm.controls.text.valid">
+                        <span translate>Required</span>
+                    </mat-hint>
+                </mat-form-field>
+            </p>
+        </form>
+    </mat-card-content>
+    <mat-card-actions>
+        <button mat-button (click)="create()" translate>Create</button>
+        <button mat-button (click)="statuteParagraphToCreate = null" translate>Abort</button>
+    </mat-card-actions>
+</mat-card>
+<mat-accordion class="os-card">
+    <mat-expansion-panel *ngFor="let statuteParagraph of this.statuteParagraphs" (opened)="openId = statuteParagraph.id"
+        (closed)="panelClosed(statuteParagraph)" [expanded]="openId === statuteParagraph.id" multiple="false">
+        <mat-expansion-panel-header>
+            <mat-panel-title>
+                {{ statuteParagraph.title }}
+            </mat-panel-title>
+        </mat-expansion-panel-header>
+        <form [formGroup]="updateForm" *ngIf="editId === statuteParagraph.id" (keydown)="keyDownFunction($event, statuteParagraph)">
+            <span translate>Edit statute paragraph details:</span>
+            <p>
+                <mat-form-field>
+                    <input formControlName="title" matInput placeholder="{{'Title' | translate}}" required>
+                    <mat-hint *ngIf="!updateForm.controls.title.valid">
+                        <span translate>Required</span>
+                    </mat-hint>
+                </mat-form-field>
+            </p>
+            <p>
+                <mat-form-field>
+                    <textarea formControlName="text" matInput placeholder="{{'Statute paragraph' | translate}}" required></textarea>
+                    <mat-hint *ngIf="!createForm.controls.text.valid">
+                        <span translate>Required</span>
+                    </mat-hint>
+                </mat-form-field>
+            </p>
+        </form>
+        <ng-container *ngIf="editId !== statuteParagraph.id">
```

Comment:
But a component more to render. If we do not explicity need a wrapping element, we can use ng-container (because this is what ngIf internally does in the wrapping element.)